apps/src/lib/cli.rs

```diff
@@ -3517,13 +3520,15 @@ pub mod args {
             let source = TRANSFER_SOURCE.parse(matches);
             let target = TRANSFER_TARGET.parse(matches);
             let token = TOKEN.parse(matches);
+            let trace_path = TRACE_PATH.parse(matches);
```

Comment:
adding a `trace_path` argument to `TxTransfer` kind of leaks ibc logic into something that should be generic. is it possible to implement this purely CLI-side, like we have for the Ethereum bridge, when parsing Ethereum addresses? this doesn't bode too well with the fact that you can specify a `token` with an IBC trace path already, then add another trace path on top of it, ending up with something like: `port/chan/token/port/chan/token`.

if you feel like that would be too disruptive to change into, we can leave it as is

Reply:
Thank you! That also works for non-Namada tokens.